### DISCUSSION TITLE
Running GROOPS with multiple processes

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#93] [Q&A] [answered]
Hello

We are planning to run GROOPS on a 32 proceessor HPC machine. Is there an option to increase from 4 processes in the GROOPS settings to use this higher capability machine?

Txs for the feedback

Cheers
Jean

**Comment by tmayerguerr** [comment, marked as answer]
Just change the 4 to 32 :-)
in menu->settings->commands. Or add just an additional command there.
Torsten
